docs/ref/django-admin.txt

```diff
@@ -1222,7 +1222,7 @@ For example::
 .. django-admin-option:: --template TEMPLATE
 
 Provides the path to a directory with a custom app template file or a path to a
-compressed file (``.tar.gz``, ``.tar.bz2``, ``.tgz``, ``.tbz``, ``.zip``)
```

Comment:
I don't think that this change is necessary because in the next commit (9ece0f6ec2638d40f1764c9e10e49216687b286f) you added `.tbz` and `.tz2` as aliases of `.tar.bz2`. I would rather remove this commit and add all missing aliases to the `docs/ref/django-admin.txt` as a part of (_" Refs #30160 -- Simplified archive extension map and add other aliases. "_).